### PR TITLE
Fix spacewalk-branding rpm

### PR DESCRIPTION
**Pull request by mbussolotto**
## What does this PR change?

Fix spacewalk-branding rpm
I tried these changes here https://build.opensuse.org/package/show/home:mbussolotto:branches:systemsmanagement:Uyuni:Master/spacewalk-branding

## GUI diff

No difference.

- [ ] **DONE**

## Documentation
- No documentation needed
- [x] **DONE**

## Test coverage
- No tests
- [x] **DONE**

## Links

- [x] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

**Review by mcalmer — APPROVED** [review]

**Comment by mbussolotto** [comment]
@deneb-alpha @raulillo82 this is the fix for the current `spacewalk-branding` rpm broken, can you please merge it :)  ? Thanks

**Comment by sbluhm** [comment]
> FYI @sbluhm everything seems to be ok also for EL9, but it there's any problem let me know.

All fine. Sorry about this one. Kept the wrong line...

File: branding/spacewalk-branding.changes.mbussolotto.fix_branding_rpm [hunk 1]
```diff
@@ -0,0 +1 @@
+- fix spacewalk-branding rpm
```
Comment (deneb-alpha) [inline branding/spacewalk-branding.changes.mbussolotto.fix_branding_rpm:1]:
```suggestion
- Fix spacewalk-branding wwwdocroot path
```

**Review by deneb-alpha — CHANGES_REQUESTED** [review]
I left a suggestion for the changelog

**Comment by mbussolotto** [comment]
@deneb-alpha Thanks :) . Sorry I thought the branch was locked cause `Merging is blocked` , but I think it's because review from code owner is missing.

**Review by Etheryte — APPROVED** [review]

**Comment by mbussolotto** [comment]
@deneb-alpha now it's blocked from your request :)

**Review by deneb-alpha — APPROVED** [review]
LGTM, thanks

**Comment by sbluhm** [comment]
@mbussolotto 
Seems like we need one more loop.
Package `spacewalk-branding` now conflicts with `spacewalk-html`
`spacewalk-branding` (line 88 or so):
` %attr(775,tomcat,tomcat) %dir %{wwwdocroot}` 

´spacewalk-html`:
`%dir %{www_path}`

You want me to remove or do you prefer to do so?


Maybe the branding should have all the tomcat ownership removed?

**Comment by mbussolotto** [comment]
where do you see this problem? At packaging time or during installation? If you know hot to fix it please go ahead and create the PR :) .

I have an installation before this problem and permission for `/usr/share/susemanager/www/htdocs` are root:root. I'd keep the tomcat ownership everywhere besides htdocs folder

**Comment by sbluhm** [comment]
It is during install time.

` ` ` 
# dnf install https://download.copr.fedorainfracloud.org/results/sbluhm/uyuni-master/epel-9-x86_64/06691238-spacewalk-branding/spacewalk-branding-4.4.1-1.git.15918.85c24f9.x86_64.rpm
Last metadata expiration check: 0:00:32 ago on Fri 24 Nov 2023 02:13:14 PM CET.
spacewalk-branding-4.4.1-1.git.15918.85c24f9.x86_64.rpm                                                                                                                                                        64 kB/s |  16 kB     00:00    
Dependencies resolved.
==============================================================================================================================================================================================================================================
 Package                                                     Architecture                                    Version                                                              Repository                                             Size
==============================================================================================================================================================================================================================================
Downgrading:
 spacewalk-branding                                          x86_64                                          4.4.1-1.git.15918.85c24f9                                            @commandline                                           16 k

Transaction Summary
==============================================================================================================================================================================================================================================
Downgrade  1 Package

Total size: 16 k
Is this ok [y/N]: y
Downloading Packages:
Running transaction check
Transaction check succeeded.
Running transaction test
Error: Transaction test error:
  file /usr/share/susemanager/www/htdocs from install of spacewalk-branding-4.4.1-1.git.15918.85c24f9.x86_64 conflicts with file from package spacewalk-html-4.4.12-1.git.15873.5c7cf8f.noarch
` ` ` 

I just created the change. Will test it on a new environment and then create a PR.
https://github.com/uyuni-project/uyuni/compare/master...sbluhm:uyuni:branding-conflict

**Comment by sbluhm** [comment]
https://github.com/uyuni-project/uyuni/pull/7954

**Pull request merged**